build-ps/debian/percona-xtradb-cluster-server-5.7.postinst

```diff
@@ -19,6 +19,7 @@ invoke() {
 }
 
 MYSQL_BOOTSTRAP="/usr/sbin/mysqld --bootstrap --user=mysql --skip-grant-tables"
+MYSQL_INIT="/usr/sbin/mysqld --initialize --user=mysql --datadir=/var/lib/mysql"
```

Comment:
Later in the file we have "mysql_install_db" used and in different place, so not sure why two different stuff in two different places. And isn't for 5.7 "mysqld --initialize" the preffered way...